ws/filetransfer/model.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package filetransfer
+
+const (
+	MessageTypeGetFile          = "get_file"
+	MessageTypePutFile          = "put_file"
+	MessageTypeStatFile         = "file_info_req"
+	MessageTypeStatFileResponse = "file_info_res"
+	MessageTypeFileChunk        = "file_chunk"
```

Comment:
I think we can remove most of the "file" prefix and suffixes from the message types and structs.
```suggestion
	MessageTypeGet       = "get"
	MessageTypePut       = "put"
	MessageTypeStat      = "stat"
	MessageTypeFileInfo  = "file_info"
	MessageTypeFileChunk = "chunk"
```

Reply:
ok, lets make a deal, I will remove from the identifiers, but leave for debugging in the values. ok?